availability.php

```diff
@@ -76,6 +76,6 @@
     <form action="availability.php" method="post">
       ' . daily_matrix ( $date, $users ) . '
     </form>
-    ' . print_trailer ( false, true, true );
+    ' . print_trailer ( 0, 1, 1 );
```

Comment:
Using "false" provides more context to the developer than "0".  The 0 and false should be identical to the end user, right?  It might be a tiny bit slower to parse on anything earlier than PHP 5.5.  According this [this](https://www.acquia.com/blog/php-getting-faster), PHP 5.5 and later essentially compile and cache the compiled PHP.

I'm okay with larger source files (adding comments, etc.) if it provides better information to other developers.